openSUSE/open-build-service
# Use component to delete projects

Use DeleteConfirmationDialogComponent instead of a partial to render the confirmation dialog.

To test this

- log in as Admin
- create a project
- go to the overview page of the project
- click on the "Delete project" link on the Actions menu (left-hand side in desktop size)
- check if the confirmation dialog contains the name of the project
- click on the "Remove" button
- go to the "All projects" page or search by the project's name to check that it no longer exists.
